### PR TITLE
Document `show_host_usage` configuration option in README

### PR DESCRIPTION
**Pull request by pinksi**
This PR updates the readme file and add the information about the custom configuration feature to show or hide the information in sidebar or tachometer showing Kernel and host usage information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,12 @@ Kernel resource usage can be displayed in a sidebar for IPython kernels with
 
 The kernel usage is available for Notebook 7.x too which can be enabled at
 `View -> Right Sidebar -> Show Kernel Usage`. In the case of JupyterLab interface, it is
-enough to click `tachometer` icon on the right sidebar.
+enough to click `tachometer` icon on the right sidebar. 
+
+If you want to show or hide information from the Kernel Usage sidebar
```
Comment (krassowski) [inline README.md:31]: Given the example sets it to `False`maybe just:

```suggestion
If you want to hide information from the Kernel Usage sidebar
```

File: README.md [hunk 2]
```diff
@@ -143,6 +148,11 @@ causes "lag"/pauses in the UI. To workaround this you can disable Prometheus met
 --ResourceUseDisplay.enable_prometheus_metrics=False
 ```
 
+### Show or Hide information from sidebar (tachometer)
+If you want to show or hide information from the Kernel Usage sidebar
```
Comment (krassowski) [inline README.md:152]: The same suggestion on wording here. Did it need to be reported twice or could one section link to the other?

Comment (pinksi) [reply README.md:152]: yes, I think once should be enough.

**Review by krassowski — APPROVED** [review]
Thank you @pinksi!

**Pull request merged**